targets/drive/mobile/fastlane/metadata/android/fr-FR/full_description.txt

```diff
@@ -1,11 +1,16 @@
-Cette application n'est compatible qu'avec la dernière version de Cozy (v3).
+éunit. Dans votre Cozy, vous mettez toutes vos données personnelles : photos, santé, bancaire, vacances, alors inaccessibles aux autres et aux entreprises.
```

Comment:
The line begins with __éunit.__ and it seems weird.

Reply:
I is :/